### DISCUSSION TITLE
[[[Links]]] are not linking, they remain raw text

### DISCUSSION THREAD

**Discussion by applejag** [srid/neuron#533] [General]
This worked before, and I'm kind of guessing it could be because of your PR #529 that you merged just recently.

I just deployed and let the GitHub actions (as taken from this template: https://github.com/srid/neuron-template) do its thing. It seems to have used the latest version and that supposedly broke something.

## Expected

> ![image](https://user-images.githubusercontent.com/2477952/105242631-7dcc4000-5b6e-11eb-8be3-949b751cf74a.png)

## Actual

> ![image](https://user-images.githubusercontent.com/2477952/105242514-64c38f00-5b6e-11eb-999d-0f1faf57e5a9.png)

Also my heterarchy is completely flat because of this.

Site: https://wiki.jillejr.tech/
Source: https://github.com/jilleJr/wiki

---

I don't plan on getting a gab account, so I'm sneaking in this question here:

Why the removal of the search? Was it only for performance reasons on large zettelkastens? And is there any plan on bringing it back, though remade (for better performance)?

**Comment by srid** [comment]
From https://github.com/jilleJr/wiki/blob/fa5320490ad78e231c5e663ef995eebc82da1b18/neuron.dhall#L8 you have only `neuronignore` activated. You should add the `links` plugin to it, as well as the `uptree` plugin. These are enabled by default. But since your neuron.dhall specified the plugins explicitly, the default is not in effect. See https://neuron.zettel.page/plugins.html

> Why the removal of the search? Was it only for performance reasons on large zettelkastens? And is there any plan on bringing it back, though remade (for better performance)?

Yea, the JS performance has been pretty bad if your notebook is large. I'm hoping to bring it back as part of the revamp of the neuron server (cf. https://github.com/srid/neuron/discussions/200 ) - so search happens on the neuron backend with the results html rendered before being served.
